assignments/assignment_5_md_version.md

```diff
@@ -0,0 +1,81 @@
+```python
+"""This is just for you so that you can see where the numbers are coming from.
+It should be downloaded with the jupyter notebook, separately each file."""
+np.random.seed(12345)
+
+cars = np.random.randint(500,8000,size=500)
+np.save('./cars.npy',cars)
+
+value = np.random.uniform(size=500)
+np.save('./value.npy',value)
+```
+
+# Assignment 5
+
+Dan a university student wants to buy a car. He knows the price of each he is
+considering and he assigned a value per pound ratio to each of the cars (based
+on horsepower, consumption etc.). He saved this data in two arrays (clearly, why
+wouldn't he?), where the order of the cars was the same. Obviously Dan doesn't
+want a slow car, he wants the car to have at least $100$ horsepower, which start
+from $1500$ pounds. So Dan did the following:
+
+```python
+# Import numpy
+import numpy as np
+```
+
+First he loaded the arrays he created earlier, in the variable called cars and
+value using a numpy function
+[np.load()](https://docs.scipy.org/doc/numpy-1.15.0/reference/generated/numpy.load.html).
```

Comment:
I get why you would want to teach them to load in, say, .csv files, but is the generating, saving and loading of a numpy array really needed? Do people often do that?

Reply:
I don't think so, I never use it, but the point was to skip that part and they should not be bothered by how is it done (it is not the point of the assessment). As it is done by us, it could be either done through reading lines from txt, read_csv or whatever, but I thought it is just nice and easy to use it in this case.

Reply:
Why not just have
```
np.random.seed(12345)
cars = np.random.randint(500,8000,size=500)
value = np.random.uniform(size=500)
```
And then the values are already stored and they don't have to read about saving and loading any files.

Reply:
What I would do - if possible, is make - say - 30 values, get numpy to show them to 2 decimals with `np.set_printoptions(precision=2)`, and then copy paste those, something like:

```
cars = np.array([5360,  558, 2749, 6549, 2814,  587,  501, 5800, 3670, 4549, 6980,
       7622, 6242, 6674, 6159, 6924, 1457, 7808, 1585, 3591, 7534, 4274,
       4751, 3182,  656, 1686, 2303, 2317, 4576, 1740])
value = np.array([0.67, 0.66, 0.48, 0.09, 0.27, 0.64, 0.55, 0.77, 0.24, 0.19, 0.19,
       0.46, 0.03, 0.48, 0.61, 0.84, 0.23, 0.8 , 0.55, 0.81, 0.61, 0.3 ,
       0.87, 0.83, 0.97, 0.56, 0.15, 0.75, 0.57, 0.33])
```

Reply:
Updated.